extensions/typescript-language-features/src/languageFeatures/quickFix.ts

```diff
@@ -384,7 +403,7 @@ function isPreferredFix(
 			return true;
 		}
 
-		if (otherAction.isFixAll) {
+		if (otherAction instanceof VsCodeFixAllCodeAction) {
```

Comment:
This and the above reference are the only usage of the isFixAll property on the VsCodeCodeAction class.  Removing these makes the property redundant.  Thus, it can be removed.  However, it is better to encapsulate this logic within the class rather than having to use instanceOf.  So, why not use the isFixAll and set the value to false in the super call within the  VsCodeFixAllCodeAction constructor? (4 parameter)